public/content/documentation/native/controls/checkbox.md

```diff
@@ -0,0 +1,295 @@
+## General Notes
+
+How to test a checkbox
+
+## Videos
+
+### iOS VoiceOver
+<video controls>
+  <source src="media/video/native/checkbox/checkboxVoiceOver.mp4" type="video/webm">
+  Your browser does not support the video tag.
+</video>
+
+### Andriod Talkback
+<video controls>
+  <source src="media/video/native/checkbox/checkboxTalkback.mp4" type="video/webm">
+  Your browser does not support the video tag.
+</video>
+
+## Condensed
+
+### #a11y - Native Accessibility Acceptance Criteria
+
+How to test a checkbox
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Tab, arrow keys or ctl+tab: Focus visibly moves to the checkbox
+   - Spacebar: Activates on iOS and Android
+   - Enter: Activates on Android
+
+2. Test mobile screenreader gestures
+
+   - Swipe: Focus moves to the element, expresses its name, role, state
+   - Doubletap: Checkbox toggles between checked and unchecked states
+
+3. Listen to screenreader output on all devices
```

Comment:
Will we be adding "Scenario" to this one or removing "Scenario" from the other three?